presto-verifier/pom.xml

```diff
@@ -45,6 +45,11 @@
             <artifactId>presto-hive-common</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-client</artifactId>
+        </dependency>
+
```

Comment:
What do we need from the presto-client in the verifier?

Reply:
One place is the following lines of change. The change is about passing the query_id from xdb to the UI by setting them in the related data structures.

In presto-verifier
```
import com.facebook.presto.client.StatementStats;
...
public static QueryActionStats queryIdStats(String queryId)
    {
        return new QueryActionStats(Optional.of(QueryStats.create(queryId, StatementStats.builder().setState(QueryState.FINISHED.toString()).build())), Optional.empty());
    }
```